src/Aquifer.API/Endpoints/Resources/Content/Get/Response.cs

```diff
@@ -178,4 +178,6 @@ public class MachineTranslationResponse
     public bool ImproveClarity { get; set; }
     public bool ImproveTone { get; set; }
     public bool ImproveConsistency { get; set; }
+    public string? RetranslationReason { get; set; }
```

Comment:
Do we need to send the reason, or do we just need to send that it was already done? I'm wondering if this should be a bool that's `HadRetranslation` and the check is that `RetranslationReason` isn't null.